app/models/user.rb

```diff
@@ -167,7 +167,7 @@ def days_since_last_clicked
   end
 
   def days_since_last_email
-    last_sent_at = repo_subscriptions.where("last_sent_at is not null").order(:last_sent_at).first.try(:last_sent_at)
+    last_sent_at = repo_subscriptions.where.not(last_sent_at: nil).order(:last_sent_at).first.try(:last_sent_at)
```

Comment:
Nice! How about using `&.` instead of `try`? It's shorter, faster (even though not a problem here) and ruby standard.

Reply:
Sure.